controllers/articles.controller.js

```diff
@@ -33,3 +34,26 @@ exports.getComments = (request, response, next) => {
     })
     .catch((err) => next(err));
 };
+
+exports.addComment = (request, response, next) => {
+  const { article_id } = request.params;
+  const { username, body } = request.body;
+  console.log(request.params);
+  console.log(request.body);
+  postComment(article_id, username, body)
+    .then(({ comment }) => {
+      response.status(201).send(comment);
+    })
+    .catch((err) => next(err));
+};
+
+exports.addComment = (request, response, next) => {
```

Comment:
You seem to have two of the same functions - this bottom one i think is the one you want